frontend/app/views/joiner/thankyou.scala.html

```diff
@@ -79,10 +79,8 @@ <h2 class="page-section__headline">@title</h2>
 </script>
     <main role="main" class="page-content l-constrained">
 
-        <!-- Conversion Pixel - Jan17 - Partner Conversion pixel - DO NOT MODIFY -->
-            <img src="https://secure.adnxs.com/px?id=@{AppnexusPixel.thankYouPageids(member.subscription.plan.tier)}&t=2" width="1" height="1" />
-        <!-- End of Conversion Pixel -->
-
+        @fragments.analytics.appNexusPixel("id", AppnexusPixel.thankYouPageids(member.subscription.plan.tier))
```

Comment:
see sentry here https://sentry.io/the-guardian/membership/issues/218738238/#
cc @svillafe